app/scripts/templates/settings/account_recovery/recovery_key_print.mustache

```diff
@@ -0,0 +1,7 @@
+<html>
+<head><title>Recovery Key</title>
+</head>
+<body>
+    <p>{{recoveryKey}}</p>
```

Comment:
Since the user has to manually enter the recovery code, I'd split this into sections of 4 as well.